display_vk/README.md

```diff
@@ -0,0 +1,3 @@
+# display_vk
+## Summary
+`display_vk` defines a service called `display_vk` which inherits from `display_sink`. The `display_vk` class is responsible for setting up and managing a Vulkan-based display sink. The `setup_vk()` function initializes GLFW and Vulkan, while the `recreate_swapchain()` function recreates the Vulkan swapchain. The `setup_vk()` function creates a Vulkan instance using the `vkb::InstanceBuilder` class from the `vkb` library. The instance is configured with validation layers and a debug callback function.
```

Comment:
Add minor details here like - 
1. is this needed for native or monado or both? if this is needed by only one, then why? What was the reasoning behind this class/inheritance etc.
2. Why do we need to recreate Vulkan Swapchain in this case